Dockerfile.rhel7

```diff
@@ -1,8 +1,27 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12 AS builder
+
+RUN dnf install -y git make go
+
 WORKDIR /go/src/github.com/openshift/must-gather
 COPY . .
 ENV GO_PACKAGE github.com/openshift/must-gather
 
+# performance profile controller gather-sysinfo binary build
+ARG REPO=github.com/marioferh/gather-sysinfo
+WORKDIR /root
+RUN git clone https://$REPO
+WORKDIR /root/gather-sysinfo 
+RUN make dist-gather-sysinfo
+
 FROM registry.ci.openshift.org/ocp/4.12:cli
 COPY --from=builder /go/src/github.com/openshift/must-gather/collection-scripts/* /usr/bin/
 RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
+RUN yum install -y pciutils util-linux hostname rsync tar ethtool
```

Comment:
This should come before the `yum clean all && rm -rf /var/cache/yum/*` parts above.

Reply:
done

Reply:
It's still not correct you should just add your tools into the original line like so:
```
RUN yum install --setopt=tsflags=nodocs -y jq pciutils util-linux hostname rsync tar ethtool && yum clean all && rm -rf /var/cache/yum/*
```

Reply:
done